solidity/contracts/converter/BancorConverterUpgrader.sol

```diff
@@ -1,12 +1,14 @@
 pragma solidity ^0.4.21;
 import './interfaces/IBancorConverterFactory.sol';
 import '../utility/Owned.sol';
+import '../utility/interfaces/IContractFeatures.sol';
 import '../utility/interfaces/IWhitelist.sol';
 
 /*
     Bancor converter dedicated interface
 */
 contract IBancorConverter is IOwned {
+    uint256 public constant FEATURE_CONVERSION_WHITELIST = 1 << 0;
```

Comment:
I think we should just rename this contract, import IBancorConverter and inherit from it instead of duplicating all these functions